src/Form/EntityViewDisplayEditForm.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\fb_instant_articles\Form;
+
+use Drupal\field_ui\Form\EntityViewDisplayEditForm as CoreEntityViewDisplayEditForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+
+/**
+ * Extends the core EntityViewDisplayEditForm to support multiple regions.
+ */
+class EntityViewDisplayEditForm extends CoreEntityViewDisplayEditForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTableHeader() {
```

Comment:
I see. The thing is that the regions column is implicit in where the field is dragged to, it sits under a clearly labeled region name. If a user wanted to use the drop downs to manage it, they can click the "Show Row Weights" link at the top right of the table, and the hidden columns all show up. I think we should leave it like that, it's a common pattern throughout these forms. OK if I revert it?